hera_corr_cm/hera_corr_cm.py

```diff
@@ -253,36 +253,54 @@ def get_f_status(self):
         Keys of returned dictionaries are snap hostnames. Values of this dictionary are
         status key/val pairs.
 
+        For the conv_info dictionary, the format is:
+            key: name of the variable in the return dictionary from this method
+            tuple:  (redis key name, conversion method from redis to this method).
+
         These keys are:
-            pmb_alert (bool) : True if SNAP PSU controllers have issued an alert. False otherwise.
-            pps_count (int)  : Number of PPS pulses received since last programming cycle
-            serial (str)     : Serial number of this SNAP board
-            temp (float)     : Reported FPGA temperature (degrees C)
-            uptime (int)     : Multiples of 500e6 ADC clocks since last programming cycle
+            is_programmed (bool): True if the host is programmed
+            adc_is_configured (bool): True if the host adc is configured
+            is_initialized (bool): True if host is initialized
+            dest_is_configured (bool): True if dest_is_configured
+            version (str)      : Version of firmware installed
+            sample_rate (float): Sample rate in MHz
+            input (str)        : comma-delimited list of stream inputs, e.g. adc,adc,adc,adc,adc,adc
```

Comment:
what does this look like for digital noise?

Reply:
Found and added.

Reply:
thanks!